doc/stdenv/meta.chapter.md

```diff
@@ -148,6 +148,36 @@ The NixOS tests are available as `nixosTests` in parameters of derivations. For
 }
 ```
 
+Alternatively, you can specify other derivations as tests. You can make use of
+the optional parameter (here: `self`) to inject the correct package without
+relying on non-local definitions, even in the presence of `overrideAttrs`. This
+means `(mypkg.overrideAttrs f).tests` will be as expected, as long as the
```

Comment:
```suggestion
means `(mypkg.overrideAttrs f).passthru.tests` will be as expected, as long as the
```

Reply:
Thanks! These are the same attributes, but your change makes the text easier to understand.

Reply:
Ah, I didn't even realize that :) .